public/Javascript/script.js

```diff
@@ -0,0 +1,310 @@
+  } else if (
+    cellFour.textContent === "X" &&
+    cellFive.textContent === "X" &&
+    cellSix.textContent === "X"
+  ) {
+    gameSatisfied();
+    markWin([cellFour, cellFive, cellSix])
+  } else if (
+    cellSeven.textContent === "X" &&
+    cellEight.textContent === "X" &&
+    cellNine.textContent === "X"
+  ) {
+    gameSatisfied();
+    markWin([cellSeven, cellEight, cellNine])
+  } else if (
+    cellOne.textContent === "X" &&
+    cellFour.textContent === "X" &&
+    cellSeven.textContent === "X"
+  ) {
+    gameSatisfied();
+    markWin([cellOne, cellFour, cellSeven])
+  } else if (
+    cellTwo.textContent === "X" &&
+    cellFive.textContent === "X" &&
+    cellEight.textContent === "X"
+  ) {
+    gameSatisfied();
+    markWin([cellTwo, cellFive, cellEight])
+  } else if (
+    cellThree.textContent === "X" &&
+    cellSix.textContent === "X" &&
+    cellNine.textContent === "X"
+  ) {
+    gameSatisfied();
+    markWin([cellThree, cellSix, cellNine])
+  } else if (
+    cellOne.textContent === "X" &&
+    cellFive.textContent === "X" &&
+    cellNine.textContent === "X"
+  ) {
+    gameSatisfied();
+    markWin([cellOne, cellFive, cellNine])
+  } else if (
+    cellThree.textContent === "X" &&
+    cellFive.textContent === "X" &&
+    cellSeven.textContent === "X"
+  ) {
+    gameSatisfied();
+    markWin([cellThree, cellFive, cellSeven])
+  } else if (
+    cellOne.textContent === "O" &&
+    cellTwo.textContent === "O" &&
+    cellThree.textContent === "O"
+  ) {
+    gameSatisfied();
+    markWin([cellOne, cellTwo, cellThree])
+  } else if (
+    cellFour.textContent === "O" &&
+    cellFive.textContent === "O" &&
+    cellSix.textContent === "O"
+  ) {
+    gameSatisfied();
+    markWin([cellFour, cellFive, cellSix])
+  } else if (
+    cellSeven.textContent === "O" &&
+    cellEight.textContent === "O" &&
+    cellNine.textContent === "O"
+  ) {
+    gameSatisfied();
+    markWin([cellSeven, cellEight, cellNine])
+  } else if (
+    cellOne.textContent === "O" &&
+    cellFour.textContent === "O" &&
+    cellSeven.textContent === "O"
+  ) {
+    gameSatisfied();
+    markWin([cellOne, cellFour, cellSeven])
+  } else if (
+    cellTwo.textContent === "O" &&
+    cellFive.textContent === "O" &&
+    cellEight.textContent === "O"
+  ) {
+    gameSatisfied();
+    markWin([cellTwo, cellFive, cellEight])
+  } else if (
+    cellThree.textContent === "O" &&
+    cellSix.textContent === "O" &&
+    cellNine.textContent === "O"
+  ) {
+    gameSatisfied();
+    markWin([cellThree, cellSix, cellNine])
+  } else if (
+    cellOne.textContent === "O" &&
+    cellFive.textContent === "O" &&
+    cellNine.textContent === "O"
+  ) {
+    gameSatisfied();
+    markWin([cellOne, cellFive, cellNine])
+  } else if (
+    cellThree.textContent === "O" &&
+    cellFive.textContent === "O" &&
+    cellSeven.textContent === "O"
+  ) {
+    gameSatisfied();
+    markWin([cellThree, cellFive, cellSeven])
+
+  }
+}
```

Comment:
This is a decent way to start this functionality and it works 100% correctly. If you wanted to save yourself a bunch of lines of code, you could refactor this by creating an array of all of the win conditions and iterate over them to check if the content of each of the win conditions are equal